.github/workflows/openms-ci.yml

```diff
@@ -352,3 +352,122 @@ jobs:
           OPENMP: "On"
           USE_STATIC_BOOST: "On"
           BUILD_FLAGS: "-j2"
+
+  build-lnx-clang:
+    runs-on: ubuntu-20.04
+    continue-on-error: true
+
+    steps:
+    # Cancels older builds if still running
+    - uses: rokroskar/workflow-run-cleanup-action@master
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master' && github.ref != 'refs/heads/develop'"
+
+    - uses: actions/checkout@v3
+      name: Checkout sources
+      with:
+        path: OpenMS
+
+    - name: Extract branch name
+      if: github.event_name == 'push'
+      shell: bash
+      run: |
+        sudo apt-get install -y jq
+        ref=`jq --raw-output .ref "$GITHUB_EVENT_PATH"`
+        ref=${ref#"refs/heads/"}
+        ref=${ref#"refs/"}
+        ref=${ref//\//-}
+        echo "RUN_NAME=$ref" >> $GITHUB_ENV
+      id: extract_branch
+
+    - name: Extract PR number
+      if: github.event_name == 'pull_request'
+      shell: bash
+      run: |
+        sudo apt-get install -y jq
+        echo "RUN_NAME=pr$(jq --raw-output .pull_request.number $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
+      id: extract_pr
+
+      #  open step log to get ssh access
+      #  If the CI has failed and the branch is ci-debug then start a tmate
+      #  session. SSH rendezvous point is emited continuously in the job output.
+      #  hack needed to fix apt dependency
+#    - name: "Debug: Package dependancies for tmate"
+#      run: |
+#        sudo apt-get install -y xz
+#        ln -s /bin/true /bin/apt-get
+#
+#    - name: "Debug: Start tmate"
+#      uses: mxschmitt/action-tmate@v3.3
+#      with:
+#        limit-access-to-actor: true
+#        sudo: false
+
+    - name: Install Xvfb package
+      run: |
+            sudo apt-get install -y xvfb
+
+    - name: Add THIRDPARTY
+      shell: bash
+      run: |
+          # initialize THIRDPARTY
+          cd OpenMS
+          git submodule update --init THIRDPARTY
+          cd ..
+          # add third-party binaries to PATH
+          # use flat THIRDPARTY structure
+          mkdir -p _thirdparty
+          cp -R OpenMS/THIRDPARTY/Linux/64bit/* _thirdparty/
+          cp -R OpenMS/THIRDPARTY/All/* _thirdparty/
+          # add third-party binaries to PATH
+          for thirdpartytool in ${{ github.workspace }}/_thirdparty/*
+          do
+            echo $thirdpartytool >> $GITHUB_PATH
+          done
+          
+    - name: Install LLVM and Clang
+      uses: KyleMayes/install-llvm-action@v1.6.0
+      with:        
+        ubuntu-version: "20.04"
+        version: "12.0.0"        
+  
+    - name: Install dev tools
+      run: |
+           sudo add-apt-repository universe
+           sudo apt update    
+           sudo apt-get install build-essential cmake autoconf patch libtool git automake
+           sudo apt-get install qtbase5-dev libqt5svg5-dev libqt5opengl5-dev
```

Comment:
OpenGL but no SQL? Weird that the tests even pass.

Reply:
I can amend. 
BTW not all tests pass (three, probably related to the SVM lib fail)

Reply:
hmm don't know I will add the packages and also to the install instructions.